src/chw.js

```diff
@@ -551,7 +548,8 @@ go.app = function() {
                                     self.user.extra.ussd_sessions = '0';
                                     return Q.all([
                                         self.im.contacts.save(self.user),
-                                        self.im.contacts.save(self.contact)
+                                        self.im.contacts.save(self.contact),
+                                        go.utils.adjust_percentage_registrations(self.im, self.metric_prefix)
```

Comment:
@imsickofmaps I moved the `adjust_percentage_registrations` here to minimise `.then` usage, but it seperates the meaning a bit - let me know what you think.